docs/pages/custom-builds/schema.mdx

```diff
@@ -1200,9 +1200,9 @@ build:
 
 #### `eas/find_and_upload_build_artifacts`
 
-> **Warning** You can currently upload each artifact type only once per workflow.
+> **Warning** **You can currently upload each artifact type only once per workflow.**<br />If you use [`eas/find_and_upload_build_artifacts`](#easfind_and_upload_build_artifacts) while having [`buildArtifactPaths`](/eas/json/#buildartifactpaths) configured in your build profile and the step finds and uploads some build artifacts, any following `eas/upload_artifact` step will fail.<br />To solve this, for now, we recommend removing `buildArtifactPaths` from custom build's profiles and uploading artifacts manually with `eas/upload_artifact` in the YAML workflow if you need to call it there.
 
-Automatically finds and uploads application archive, other build artifacts, and Xcode logs looking at default locations and respecting `${ eas.job.buildArtifactPaths }` setting. Uploads found artifacts to the EAS servers.
+Automatically finds and uploads application archive, other build artifacts, and Xcode logs looking at default locations and respecting [`buildArtifactPaths`](/eas/json#buildartifactpaths) setting. Uploads found artifacts to the EAS servers.
```

Comment:
```suggestion
Automatically finds and uploads application archive, additional build artifacts, and Xcode logs from the default locations and using the [`buildArtifactPaths`](/eas/json#buildartifactpaths) configuration. Uploads found artifacts to the EAS servers.
```

- There seems to be something missing in the first line. I rephrased the second half too since I think we want to mention that it uses `buildArtifactPaths` configuration.